docs/resources/credential_type.md

```diff
@@ -20,20 +20,23 @@ resource "pingone_environment" "my_environment" {
 }
 
 resource "pingone_credential_type" "verifiedemployee" {
-  environment_id = pingone_environment.my_environment.id
-  title          = "VerifiedEmployee"
-  description    = "PingOne Credentials Example"
-  card_type      = "VerifiedEmployee"
+  environment_id       = pingone_environment.my_environment.id
+  title                = "VerifiedEmployee"
+  description          = "Demo Proof of Employment"
+  card_type            = "VerifiedEmployee"
+  card_design_template = "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 740 480\"><rect fill=\"none\" width=\"736\" height=\"476\" stroke=\"#CACED3\" stroke-width=\"3\" rx=\"10\" ry=\"10\" x=\"2\" y=\"2\"></rect><rect fill=\"$${cardColor}\" height=\"476\" rx=\"10\" ry=\"10\" width=\"736\" x=\"2\" y=\"2\" opacity=\"$${bgOpacityPercent}\"></rect><image href=\"$${backgroundImage}\" opacity=\"$${bgOpacityPercent}\" height=\"301\" rx=\"10\" ry=\"10\" width=\"589\" x=\"75\" y=\"160\"></image><image href=\"$${logoImage}\" x=\"42\" y=\"43\" height=\"90px\" width=\"90px\"></image><line y2=\"160\" x2=\"695\" y1=\"160\" x1=\"42.5\" stroke=\"$${textColor}\"></line><text fill=\"$${textColor}\" font-weight=\"450\" font-size=\"30\" x=\"160\" y=\"90\">$${cardTitle}</text><text fill=\"$${textColor}\" font-size=\"25\" font-weight=\"300\" x=\"160\" y=\"130\">$${cardSubtitle}</text></svg>"
```

Comment:
Good suggestion.  The heredoc string _seems_ to work for readability, but additional testing is needed to ensure the template is rendering the credential properly in the digital wallet. 

I probably need to provide three examples: a heredoc example, an example reading the template from a file, and an example reading the template from a data source.

Reply:
I think we only need the heredoc example, we can leave the dev to fill the gaps.  The reason why the heredoc is the most valuable is because elements of the svg may be calculated in the terraform hcl (e.g. the card title gets a `- dev`, `- test` or `- prod` suffix)

Reply:
Implemented.